roles/kubernetes/node/defaults/main.yml

```diff
@@ -263,3 +263,8 @@ conntrack_modules:
 kubelet_tracing: false
 kubelet_tracing_endpoint: 0.0.0.0:4317
 kubelet_tracing_sampling_rate_per_million: 100
+
+# Whether pull images one at a time
+kubelet_serialize_image_pulls: true
+# The maximum number of image pulls in parallel. This option takes effect if kubelet_serialize_image_pulls is false
+kubelet_max_parallel_image_pulls: 5
```

Comment:
HI @tu1h 

The `maxParallelImagePulls` default value is nil: https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration.

So we need to comment on it by default like:
```
# kubelet_max_parallel_image_pulls: 5
```

Reply:
I updated it, please check if it is applicable.